package.json

```diff
@@ -15,6 +15,7 @@
     "webpack-merge": "^4.1.0"
   },
   "dependencies": {
+    "codec-detect": "^1.3.0",
```

Comment:
Published from my fork of the original library: https://github.com/brianpeiris/codec-detect